content/docs/latest/installing/cloud/azure.md

```diff
@@ -116,6 +116,15 @@ Flatcar Pro images were paid marketplace images that came with commercial suppor
 
 When building an image based on the Marketplace image you sometimes need to specify the original plan. The plan name is the image SKU, e.g., `stable`, the plan product is the image offer, e.g., `flatcar-container-linux-free`, and the plan publisher is the same (`kinvolk`).
 
+## Community Shared Image Gallery
+
+A public Shared Image Gallery hosts recent Flatcar Stable images for amd64. Here is how to show the image definitions (for now you will only find `flatcar-stable-amd64`) and the image versions they provide:
+
+```shell
+az sig image-definition list-community --public-gallery-name flatcar-23485951-527a-48d6-9d11-6931ff0afc2e --location westeurope
+az sig image-version list-community --public-gallery-name flatcar-23485951-527a-48d6-9d11-6931ff0afc2e --gallery-image-definition flatcar-stable-amd64 --location westeurope
```

Comment:
maybe we should only reference the CAPI images here? the marketplace scales better and has better insights into usage

Reply:
I think both, the SIG is used by people to build CAPI images at the moment.

Reply:
ok, then how about at least mentioning that using marketplace is recommended unless dealing with image-builder?